tools/topology/sof-tgl-nocodec-8ch.m4

```diff
@@ -0,0 +1,186 @@
+#	48000, 48000, 48000)
+
+# Low Latency playback pipeline 5 on PCM 2 using max 2 channels of s16le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	5, 2, 8, s16le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Low Latency capture pipeline 6 on PCM 2 using max 2 channels of s16le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
+	6, 2, 8, s16le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Passthrough capture pipeline 13 on PCM 6 using max 2 channels.
+# 1000us deadline on core 0 with priority 0
+# PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+# 	13, 6, 8, s32le,
+# 	1000, 0, 0,
+# 	48000, 48000, 48000)
+# #
+# DAIs configuration
+#
+
+dnl DAI_ADD(pipeline,
+dnl     pipe id, dai type, dai_index, dai_be,
+dnl     buffer, periods, format,
+dnl     deadline, priority, core, time_domain)
+
+# playback DAI is SSP0 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+#DAI_ADD(sof/pipe-dai-playback.m4,
+#	1, SSP, 0, NoCodec-0,
+#	PIPELINE_SOURCE_1, 2, s16le,
+#	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# capture DAI is SSP0 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+#DAI_ADD(sof/pipe-dai-capture.m4,
+#	2, SSP, 0, NoCodec-0,
+#	PIPELINE_SINK_2, 2, s16le,
+#	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# playback DAI is SSP1 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+#DAI_ADD(sof/pipe-dai-playback.m4,
+#	3, SSP, 1, NoCodec-1,
+#	PIPELINE_SOURCE_3, 2, s16le,
+#	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# capture DAI is SSP1 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+#DAI_ADD(sof/pipe-dai-capture.m4,
+#	4, SSP, 1, NoCodec-1,
+#	PIPELINE_SINK_4, 2, s16le,
+#	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# playback DAI is SSP2 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-playback.m4,
+	5, SSP, 2, NoCodec-2,
+	PIPELINE_SOURCE_5, 2, s16le,
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# capture DAI is SSP2 using 2 periods
+# Buffers use s16le format, with 48 frame per 1000us on core 0 with priority 0
+DAI_ADD(sof/pipe-dai-capture.m4,
+	6, SSP, 2, NoCodec-2,
+	PIPELINE_SINK_6, 2, s16le,
+	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+# capture DAI is DMIC 0 using 2 periods
+# Buffers use s32le format, 1000us deadline on core 0 with priority 0
+# DAI_ADD(sof/pipe-dai-capture.m4,
+# 	13, DMIC, 0, NoCodec-6,
+# 	PIPELINE_SINK_13, 2, s32le,
+# 	1000, 0, 0, SCHEDULE_TIME_DOMAIN_TIMER)
+
+dnl PCM_DUPLEX_ADD(name, pcm_id, playback, capture)
+#PCM_DUPLEX_ADD(Port0, 0, PIPELINE_PCM_1, PIPELINE_PCM_2)
+#PCM_DUPLEX_ADD(Port1, 1, PIPELINE_PCM_3, PIPELINE_PCM_4)
+PCM_DUPLEX_ADD(Port2, 2, PIPELINE_PCM_5, PIPELINE_PCM_6)
+# PCM_CAPTURE_ADD(DMIC01, 6, PIPELINE_PCM_13)
+
```

Comment:
Probably confusing here for users to uncomment the correct PCM/DAI config. Could the PCM s be next to there DAI_ADD so it's easy to uncomment the correct items when changing mode.